src/Storage/Adapter/AbstractAdapter.php

```diff
@@ -106,9 +117,10 @@ public function __destruct()
     /**
      * Set options.
```

Comment:
@laminas/technical-steering-committee We definitely need a sniff which checks if the method description is just repeating the methods name.